src/main/java/com/example/springresttask/domain/Email.java

```diff
@@ -0,0 +1,45 @@
+package com.example.springresttask.domain;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+
+@Getter
+@Setter
+@ToString
```

Comment:
Never use basic form of `@ToString` in entity.
I will explain it today on our lection.